### PR TITLE
CI: Add release workflow

### LINKED ISSUE #24: CI: Add release workflow

### PR DESCRIPTION
**Pull request by alexdewar**
This PR adds a release workflow to zip up each of the example models and upload them as release artifacts.

I also removed a couple of symlinks that were in the repo and replaced them with copies of the files they pointed to instead, because symlinks aren't guaranteed to work on Windows.

Closes #24.

### CODE REVIEW

File: KevinHall_India/residual_policy_covariance.csv
Comment (jamesturner246) [inline KevinHall_India/residual_policy_covariance.csv:1]: I think the corresponding policy covariance file also needs copying into the root project folder, as you've done for `static_model.json`.

**Review by jamesturner246 — CHANGES_REQUESTED** [review]
Looks good. Just the policy covariance comment below and all good.

File: KevinHall_India/residual_policy_covariance.csv
Comment (alexdewar) [reply KevinHall_India/residual_policy_covariance.csv:1]: I actually changed the file path (L175), so the config file will still work. I double-checked that it is still loadable with HGPS.

**Review by jamesturner246 — APPROVED** [review]
Oh yes, so you have.. My bad. Looks good then! :+1:

**Pull request merged**